src/clj/game/cards/identities.clj

```diff
@@ -93,9 +93,11 @@
                   :choices {:max 1
                             :req #(and (= (:side %) "Runner")
                                        (in-hand? %))}
-                  :req (req (> (count (:hand runner)) 0))
+                  :req (req (and (pos? (count (:hand runner)))
+                                 (:runner-phase-12 @state)))
```

Comment:
No one will miss an opportunity to fire Apex. It will give you the blue notification and the pause before taking clicks to manually trigger it if it's important to order it relative to something else. If left unclicked during that pause, it will fire when you hit "Take Clicks" (via the `:runner-turn-begins` event).